lib/rawHttpRequest/index.js

```diff
@@ -55,6 +55,20 @@ function setupRawHttpRequestOperationConfig (connectorsConfig) {
 						//No change; provide both endpoint and full
 						break;
 				}
+
+				// Add the base URL to the help text for the endpoint input property
+				// so that users know what the base URL actually is
+				const baseUrl = _.get(connectorConfig, 'globalModel.baseUrl');
+				if (_.isString(baseUrl)) {
+					const endpointDescription = `The endpoint to call in relation to the base URL ${baseUrl}`;
+					switch (schemaOptions.url) {
+						case 'endpointOnly':
```

Comment:
shouldn't there be a case for `fullOnly` with just a break, so that it is skipped - otherwise `fullOnly` will go to the default path?